src/App.js

```diff
@@ -1,24 +1,28 @@
-import logo from './logo.svg';
+import { useState } from 'react';
+import { getUser } from './services/users';
+
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import Home from './views/home/Home';
+import Auth from './views/auth/Auth';
+import Header from './components/header';
 import './App.css';
 
 function App() {
+  const [currentUser, setCurrentUser] = useState(getUser());
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <Header currentUser={currentUser} setCurrentUser={setCurrentUser} />
+      <div className="App">
+        <Switch>
+          <Route exact path="/">
+            <Home />
+          </Route>
+          <Route exact path="/auth">
+            <Auth setCurrentUser={setCurrentUser} />
+          </Route>
```

Comment:
In our routes, we will want to check for currentUser and render either the home or auth page so that non-authenticated users can't see the todo page and authenticated users cannot access the sign-in/sign-up page.